riot/lol/model.go

```diff
@@ -608,7 +608,7 @@ type CurrentGameParticipant struct {
 
 // GetChampion returns the champion played by this participant
 func (p *CurrentGameParticipant) GetChampion(client *datadragon.Client) (datadragon.ChampionDataExtended, error) {
-	return client.GetChampionByID(strconv.Itoa(p.ChampionID))
+	return client.GetChampionByKey(strconv.Itoa(p.ChampionID))
```

Comment:
let's add a comment on `CurrentGameParticipant.ChampionID`, describing that the value is actually the champion key. Also, I feel like this is a bug in the API :D